src/reflect/scala/reflect/internal/Types.scala

```diff
@@ -1194,6 +1194,7 @@ trait Types
       else super.safeToString
     override def narrow: Type = this
     override def kind = "ThisType"
+    override def boundSyms = if (sym.isInstanceOf[StubSymbol]) emptySymbolSet else super.boundSyms
```

Comment:
Does this mean the review is currently still pending? Lets make that clear with "request changes" review. I was about to merge this PR because the review status was "approved".